.github/workflows/main.yaml

```diff
@@ -285,3 +285,23 @@ jobs:
         uses: ./.github/actions/on_host_test
         with:
           os: linux
+
+  # Gets unit test report from on host tests and prints it.
+  on-host-test-report:
```

Comment:
Lets change this to `on-host-unit-test-report` to make it clear that integration tests are not yet collected here. We can change back once we have a solution for all.